mend_sca_cleanup_tool/sca_cleanup_tool.py

```diff
@@ -116,10 +133,15 @@ def delete_scan(product_token, project):
                 "projectToken": project['token'],
                 "agentInfo": AGENT_INFO
             })
-    response_obj = json.loads(post_api_request(request).decode("utf-8"))
+    r = post_api_request(request)
+    try:
+        response_obj = json.loads(r.decode("utf-8"))
+    except:
+        response_obj = json.loads(r)
```

Comment:
Why the try-except here? What would cause the api request to fail the decode?

Reply:
Which is why there should be two different methods to handle the different calls based on the config. We are only using requests when they are using a proxy correct?